Lib/logging/handlers.py

```diff
@@ -800,52 +800,54 @@ def __init__(self, address=('localhost', SYSLOG_UDP_PORT),
         Initialize a handler.
 
         If address is specified as a string, a UNIX socket is used. To log to a
-        local syslogd, "SysLogHandler(address="/dev/log")" can be used.
+        local syslogd, `SysLogHandler(address="/dev/log")` can be used.
         If facility is not specified, LOG_USER is used. If socktype is
         specified as socket.SOCK_DGRAM or socket.SOCK_STREAM, that specific
         socket type will be used. For Unix sockets, you can also specify a
         socktype of None, in which case socket.SOCK_DGRAM will be used, falling
         back to socket.SOCK_STREAM.
         """
-        logging.Handler.__init__(self)
+        super().__init__()
```

Comment:
Don't make changes to "tidy up the code base". Address only specific things which cause the issue.